web/ui/dashboard/src/app/public/login/login.service.ts

```diff
@@ -22,4 +22,29 @@ export class LoginService {
 			}
 		});
 	}
+
+	getSignupConfig(): Promise<HTTP_RESPONSE> {
+		return new Promise(async (resolve, reject) => {
+			try {
+				const response = await this.http.request({
+					url: '/configuration/is_signup_enabled',
+					method: 'get'
+				});
+
+				localStorage.setItem('IS_SIGNUP_ENABLED', JSON.stringify(response.data));
+				return resolve(response);
+			} catch (error) {
+				return reject(error);
+			}
+		});
+	}
+
+	get signupConfig() {
```

Comment:
Why do we need this?

Reply:
this is what I'm calling in the signup guard to check if signup is enabled